backend/src/services/board.service.spec.ts

```diff
@@ -70,6 +72,34 @@ describe("BoardsService", () => {
     ).rejects.toThrowError(UnauthorizedError);
   });
 
+  it("Should throw ConflictError when bot is already present on other board", async () => {
+    const boards = boardsService.getAll();
+    spyOn(botsService, "get").and.returnValue({
+      token: dummyBoardToken,
+      botName: "name",
+      email: "email",
+    } as IBot);
+    await boardsService.join(boards[0].id, dummyBoardToken);
```

Comment:
Should change dummyBoardToken to dummyBotToken. boardsService.join expects a boardId and a botToken.
`public async join(boardId: number, botToken: string)`

Reply:
Yeah you are correct!